packages/editor/src/components/block-list/style.scss

```diff
@@ -1,104 +1,40 @@
 .editor-block-list__layout .components-draggable__clone {
-	& > .editor-block-list__block > .editor-block-list__block-draggable {
-		background: $white; // @todo: ensure this works with themes that invert the color
-		box-shadow: $shadow-popover;
-
-		@include break-small {
-			left: -$block-parent-side-ui-clearance - $border-width;
-			right: -$block-parent-side-ui-clearance - $border-width;
-
-			.editor-block-list__layout & {
-				left: -$border-width;
-				right: -$border-width;
-			}
-		}
-	}
-
 	// Hide the Block UI when dragging the block
 	// This ensures the page scroll properly (no sticky elements)
 	.editor-block-contextual-toolbar,
-	.editor-block-mover,
 	.editor-block-settings-menu {
 		// I think important is fine here to avoid over complexing the selector
 		display: none !important;
 	}
 }
 
-.editor-block-list__layout .editor-block-list__block-draggable {
-	position: absolute;
-	top: 0;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	z-index: z-index(".editor-block-list__block-draggable");
-
-	> .editor-block-list__block-draggable-inner {
-		position: absolute;
-		top: 0;
-		right: 0;
-		bottom: 0;
-		left: 0;
-		visibility: hidden;
-
-		// Use opacity to work in various editor styles.
-		background-color: $dark-opacity-light-200;
-
-		.is-dark-theme & {
-			background-color: $light-opacity-light-200;
+.editor-block-list__layout .editor-block-list__block.is-selected { // Needs specificity to override inherited styles.
+	// While block is being dragged, dim the slot dragged from, and hide some UI.
+	&.is-dragging {
+		.editor-block-list__block-edit::before {
+			outline: none;
 		}
 
-		@include break-small {
-			margin: 0 48px;
+		> .editor-block-list__block-edit > * {
+			background: $light-gray-100;
 		}
-	}
 
-	&.is-visible > .editor-block-list__block-draggable-inner {
-		visibility: visible;
-	}
-
-	@include break-small {
-		// use a wider available space for hovering/selecting/dragging on top level blocks
-		left: -$parent-block-padding - $block-padding;
-		right: -$parent-block-padding - $block-padding;
-
-		// use smaller space for hovering/selecting/dragging on child blocks
-		.editor-block-list__layout & {
-			left: 0;
-			right: 0;
+		.freeform-toolbar,
```

Comment:
Why targetting a specific block. Maybe these could be moved to the `editor.scss` file of the block?